src/components/SearchResultList/FilterForm.tsx

```diff
@@ -0,0 +1,118 @@
+'use client';
+
+import * as React from 'react';
+import Select from 'react-select';
+
+import { Formik } from 'formik';
+
+import { SearchTypeOptions, SearchScopeOptions } from '@/constants/constants';
+import { SearchScope, SearchType } from '@/constants/enum';
+import { SearchResultFilter } from '@/types/query';
+
+interface InjectedProps {
+  onApplyFilter: (filterParams: SearchResultFilter) => void;
+  onResetFilter: () => void;
+}
+
+interface FormValues {
```

Comment:
Same idea about naming:

```suggestion
interface FilterFormValues {
```